scripts/devtron-reference-helm-charts/deployment-chart_4-19-0/templates/deployment.yaml

```diff
@@ -129,6 +145,10 @@ spec:
           {{- end }}
 {{- end }}
 {{- end }}
+{{- if $.Values.topologySpreadConstraint }}
+      topologySpreadConstraints: 
```

Comment:
continue the existing topologyspreadconstraints, can use the if condition where required.